superset/config.py

```diff
@@ -304,6 +304,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     # Exposes API endpoint to compute thumbnails
     "THUMBNAILS": False,
     "REDUCE_DASHBOARD_BOOTSTRAP_PAYLOAD": True,
+    "REDUCE_SLICE_LEVEL_LABEL_COLORS": False,
```

Comment:
naming nit, `REMOVE` instead of `REDUCE` probably

Reply:
thanks. fixed!